src/ripple/app/rdb/backend/RelationalDBInterfaceSqlite.cpp

```diff
@@ -243,6 +246,26 @@ class RelationalDBInterfaceSqliteImp : public RelationalDBInterfaceSqlite
         return !!txdb_;
```

Comment:
Nit: this idiom is not necessary hurts readability. The unque_ptr bool operator returns whether there's a managed object or not: http://www.cplusplus.com/reference/memory/unique_ptr/operator%20bool/

Reply:
Done.